build/jobs/provision-deploy.yml

```diff
@@ -0,0 +1,87 @@
+
+parameters:
+- name: version
+  type: string
+- name: sql
+  type: boolean
+  default: false
+- name: webAppName
+  type: string
+- name: appServicePlanName
+  type: string
+- name: appServicePlanResourceGroup
+  type: string
+- name: subscription
+  type: string
+- name: resourceGroup
+  type: string
+- name: testEnvironmentUrl
+  type: string
+- name: imageTag
+  type: string
+
+jobs:
+- job: provisionEnvironment
+  pool:
+    vmImage: $(WindowsVmImage)
+  steps:
+  - task: AzureKeyVault@1
+    displayName: 'Azure Key Vault: resolute-oss-tenant-info'
+    inputs:
+      azureSubscription: $(ConnectedServiceName)
+      KeyVaultName: 'resolute-oss-tenant-info'
+
+  - task: AzurePowerShell@4
+    displayName: 'Azure PowerShell script: InlineScript'
+    inputs:
+      azureSubscription: $(ConnectedServiceName)
+      azurePowerShellVersion: latestVersion
+      ScriptType: inlineScript
+      Inline: |
+        Add-Type -AssemblyName System.Web
+     
+        $deployPath = "$(System.DefaultWorkingDirectory)/test/Configuration"
+     
+        $cors = (ConvertFrom-Json (Get-Content -Raw "$deployPath/corstestconfiguration.json"))
+        $flattenedCors = $(System.DefaultWorkingDirectory)/release/scripts/PowerShell/ConvertTo-FlattenedConfigurationHashtable.ps1 -InputObject $cors
+     
+        $additionalProperties = $flattenedCors
+     
+        $additionalProperties["SqlServer__DeleteAllDataOnStartup"] = "true"
+        $additionalProperties["ASPNETCORE_FORWARDEDHEADERS_ENABLED"] = "true"
+
+        $templateParameters = @{
+            fhirVersion = "${{ parameters.version }}"
+            # appServicePlanResourceGroup = "${{ parameters.appServicePlanResourceGroup }}"
+            # appServicePlanName = "${{ parameters.appServicePlanName }}"
```

Comment:
Not needed now?

Reply:
Not needed currently. The shared App Service Plans don't work with the linux containers, but I think we will still want shared App Service Plans for cost reasons. I'll make new shared ones.

Reply:
Annoyingly I also had to make a new resource group as app service plans for windows and linux can't be in the same resource group and region. The shared plan is in msh-fhir-pr-linux.